src/api/middlewares/middlewares.auth.js

```diff
@@ -55,3 +55,83 @@ export const checkIfEmailAlreadyExist = async (req, res, next) => {
     logger.error(`checking existing email failed::${enums.CHECK_EXISTING_EMAIL}`, error.message);
   }
 };
+
+export const emailDoesNotExist = async (req, res, next) => {
+  try {
+    const { email } = req.body;
+    const user = await authServices.findEmail([ email ]);
+    logger.info(`${enums.CURRENT_TIME_STAMP}, :::Info: user with email found`);
```

Comment:
your logger.info should always show the name of the function and controller or middleware